frontend/src/App.tsx

```diff
@@ -1,31 +1,26 @@
 import React from 'react';
 import {BrowserRouter, Navigate, Route, Routes} from "react-router-dom";
-import {Container, CssBaseline} from "@mui/material";
+import {CssBaseline} from "@mui/material";
 import MixtapesOverviewPage from "./pages/MixtapesOverviewPage";
 import PlayMixtapePage from "./pages/PlayMixtapePage";
-import HeaderNav from "./components/HeaderNav";
-import BottomNav from "./components/BottomNav";
 import LoginPage from "./pages/LoginPage";
 import MixtapeDetailPage from "./pages/MixtapeDetailPage";
+import MainLayout from "./utils/MainLayout";
```

Comment:
MainLayout could be placed inside of components/ instead of utils/.

Reply:
Done